package.json

```diff
@@ -22,9 +22,11 @@
   },
   "homepage": "https://github.com/SBoudrias/run-async",
   "dependencies": {
+    "is-promise": "^2.1.0",
     "once": "^1.3.0"
   },
   "devDependencies": {
+    "bluebird": "^2.10.2",
```

Comment:
It's just a devDependency, so it won't impact anybody but people developing on it.

I tried `pinkie` at first, but it does not emit `unhandledRejection` events on process, so you had to hunt around for the error when it failed. Switching to `bluebird` surfaced errors immediately.

We could do [native-or-bluebird](https://www.npmjs.com/package/native-or-bluebird)